libgeopm/src/Controller.cpp

```diff
@@ -289,6 +289,12 @@ namespace geopm
         if (m_do_init_control) {
             m_init_control->parse_input(environment().init_control());
         }
+        try {
+            geopm::enable_fixed_counters(m_platform_io);
+        }
+        catch (const Exception &ex) {
+
```

Comment:
Might be work spelling out in a comment either here in the exception block or in the implementation for enable_fixed_counters that this is only necessary for platforms that support the MSRIOGroup.